### PR TITLE
Update several mode-converter designs and add line-spacing data

### PR DESCRIPTION
**Pull request by oskooi**
Updates several of the CSV files of the mode-converter designs which had not been correctly added in #32.

Following #36, I also measured the line *spacing* of all designs via `1.0-weights` and found they were equivalent to the line width. These results have been added to the `README.md`.

### CODE REVIEW

**Comment by oskooi** [comment]
Several changes bundled into the latest commit:
1. Modified the optimization script `mode_converter_meep_opt.py` to apply the projection operator within the subpixel-smoothing step via its `beta` parameter rather than using a weight mapping function which also includes a filter step. Details for this setup are provided in the Meep tutorial [Broadband Waveguide Mode Converter with Minimum Feature Size](https://meep.readthedocs.io/en/latest/Python_Tutorials/Adjoint_Solver/#broadband-waveguide-mode-converter-with-minimum-feature-size).
2. Based on the change in (1), regenerated designs with a designed minimum feature size (MFS) of 50, 60, 70, and 90 nm. The remaining designs will be updated in a separate PR as they require additional manual hyperparameter tweaking.
3. Resolved the merge conflict in `README.md`.

File: waveguide_mode_converter/README.md [hunk 1]
```diff
@@ -47,7 +47,6 @@ python3 ./run.py designs/converter_*.csv
 
 Will produce the output below:
 
-```
 # Design file, Length scale (nm), Worst-case reflection (dB), Worst-case transmission (dB)
```
Comment (ianwilliamson) [inline waveguide_mode_converter/README.md:50]: This line is _not_ meant to be markdown. It's the header of the CSV file whose contents are displayed in the code block. When copying and importing this CSV as a Pandas data frame, these headers are used to name the columns.

Comment (ianwilliamson) [reply waveguide_mode_converter/README.md:50]: Please also keep the code blocks "```"

Comment (oskooi) [reply waveguide_mode_converter/README.md:50]: Fixed.

**Pull request merged**